test/specs/plugin.filler.tests.js

```diff
@@ -10,6 +11,43 @@ describe('Plugin.filler', function() {
   describe('auto', jasmine.fixture.specs('plugin.filler'));
 
   describe('dataset.fill', function() {
+    it('Should show a warning when trying to use the filler plugin in the dataset when it\'s not registered', function() {
+      spyOn(console, 'warn');
+      Chart.unregister(Chart.Filler);
+      window.acquireChart({
+        type: 'line',
+        data: {
+          datasets: [{
+            fill: true
+          }]
+        }
+      });
+
+      expect(console.warn).toHaveBeenCalledWith(fillerPluginRegisterWarning);
+
+      Chart.register(Chart.Filler);
+    });
+
+    it('Should show a warning when trying to use the filler plugin in the root options when it\'s not registered', function() {
+      // jasmine.createSpy('warn');
+      spyOn(console, 'warn');
```

Comment:
Leftover comment. Does the `spyOn` need to be reversed / turned off?

Reply:
This automatically happens: https://github.com/jasmine/jasmine/issues/236#issuecomment-28958973